com.exok.Celeste64.yml

```diff
@@ -0,0 +1,48 @@
+id: com.exok.Celeste64
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet8
+build-options:
+  arch:
+    x86_64:
+      env:
+        RUNTIME: "linux-x64"
+# support for aarm64 will be added in the future
+#    aarch64:
+#      env:
+#        RUNTIME: "linux-arm64"
+  prepend-pkg-config-path: "/usr/lib/sdk/dotnet8/lib/pkgconfig"
+  append-path: "/usr/lib/sdk/dotnet8/bin"
+  append-ld-library-path: "/usr/lib/sdk/dotnet8/lib"
+command: celeste64
+finish-args:
+  - --device=all
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=x11
+
+modules:
+  - name: celeste64
+    buildsystem: simple
+    only-arches:
+      - "x86_64"
+#      - "aarm64" # support for aarm64 will be added in the future
+    build-commands:
+      - "install-sdk.sh"
+      - "dotnet publish -c Release --runtime $RUNTIME --source ./nuget-sources Celeste64.csproj"
+      - "cp -r --remove-destination /run/build/celeste64/bin/Release/net8.0/$RUNTIME/publish/ /app/${FLATPAK_ID}"
+      - "cp -r --remove-destination /run/build/celeste64/Content/ /app/${FLATPAK_ID}/Content"
+      - "ln -s /app/${FLATPAK_ID}/Celeste64 /app/bin/celeste64"
+      
+      - install -Dm644 .flatpak/${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm644 .flatpak/${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 .flatpak/${FLATPAK_ID}.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+
+    sources:
+      - type: "git"
+        url: "https://github.com/brunofin/Celeste64.git"
```

Comment:
Please don't switch it to a fork. You can add direct links to desktop file icon as secondary sources.